src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -285,6 +285,8 @@
     short-summary: The deployment name.
   - name: --what-if-result-format -r
     short-summary: The format of What-If results. Applicable when --confirm-with-what-if is set.
+  - name: --what-if-exclude-change-types -x
+    short-summary: Space-separated list of resource change types to be excluded from What-If results. Possible values are Create, Delete, Modify, Deploy, NoChange, and Ignore. Applicable when --confirm-with-what-if is set.
```

Comment:
This is repeated with the help declared in `deployment_what_if_exclude_change_types_type`, and the help declared in `_params.py` will override the help in `_help.py`

Reply:
Not sure if I misunderstood, but I think it's the other way it's the other way around, which is I what want. The short-summaries in `_help.py` override the help message specified in `deployment_what_if_exclude_change_types_type` - they end with `Applicable when --confirm-with-what-if is set.`.

Reply:
@shenglol I'm sorry, I typed it wrong. Doc: [help-layers](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_help.md#help-layers)
Actually, what I wanted to say was that there was no need to write help in two places, right?

Reply:
No worries at all. Good point. I moved the help messages to `_params.py`.